javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,21 @@
+const charArray = Array.from(document.querySelectorAll('span'));
```

Comment:
There's no need for `Array.from`. `querySelectorAll` is a `NodeList` which is array like, so you can just use that directly